P1 test cases for network policy

sangeethah opened the pull request:
@leodotcloud Could you review the P1 test cases for network policy?  
I will be adding standalone containers to the use cases as well.

leodotcloud commented on tests/v2_validation/cattlevalidationtest/core/test_network_policy.py:
```diff
@@ -0,0 +1,287 @@
+    'TEST_NETWORK_POLICY', "False")
+np_reason = \
+    'Rancher NetworkPolicy Manager needs to de deployed in "Default" env'
+if_network_policy = pytest.mark.skipif(test_network_policy != "True",
```
May be you need one more reason np_reason_2="NPM tests are disabled"

And then check if the network_policy_manager stack is deployed in the env before you start running the tests and use the np_reason?

sangeethah replied:
@leodotcloud , I have changed the existing reason string to - ```Intended to not execute network policy test``` . Added check_for_network_policy_manager() method to create_env_for_network_policy so that it prevents the setup and  any test from getting executed when network_policy_manager stack  is not deployed in the setup.

leodotcloud commented:
@sangeethah Except for my one comment, rest looks fine.

sangeethah commented:
@leodotcloud , I have changed the existing reason string to - Intended to not execute network policy test . Added check_for_network_policy_manager() method to create_env_for_network_policy so that it prevents the setup and any test from getting executed when network_policy_manager stack is not deployed in the setup.

leodotcloud commented:
LGTM
